pkg/pmem-csi-driver/nodeserver.go

```diff
@@ -199,14 +199,12 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 			return nil, status.Error(codes.Internal, "validate target path: "+err.Error())
 		}
 		if !notMnt {
-			// TODO(https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/issues/95):
 			// Check if mount is compatible. Return OK if these match:
 			// 1) Requested target path MUST match the published path of that volume ID
 			// 2) VolumeCapability MUST match
-			//   Note that we do not check VolumeCapability/fsType as fsType may not be present (is used in NodeStageVolume scope).
-			//   Means, we check only mountflags part of Capability.
-			//   Mount flags are coded as []string, so we sort and join mount flags to single string
-			//   for easier comparing.
+			//    We do not check VolumeCapability/fsType as fsType may be missing (is used in NodeStageVolume).
```

Comment:
These remvoed comments are added part of this PR, I feel like you can squash with your earlier commit which introduced these comments. I am ok to completely sqashing this commit too.

Reply:
thanks, yes I had changed in later commit that I had created in previous one. I now combined the changes of text-changing commit into previous bigger commit.